tsconfig.paths.json

```diff
@@ -2,8 +2,7 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@functions/*": ["src/functions/*"],
-      "@libs/*": ["src/libs/*"]
+      "@functions/*": ["src/functions/*"]
```

Comment:
Is this file needed?

Reply:
yes, the tsconfig.json needs this route, otherwise I got an error during the build process of the cow-slug stuff.